website/pages/ar/_meta.js

```diff
@@ -1,5 +1,5 @@
 import meta from '../en/_meta.js'
 
 export default {
-  ...meta,
+  ...JSON.parse(JSON.stringify(meta)),
```

Comment:
Could we use [`structuredClone()`](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) instead?